docs/apis/core/htmlwriter/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: HTML Writer API
+tags:
+  - ' API'
+  - ' HTML'
+  - ' DOM'
+---
+
+Moodle has a class called _HTML writer_ which allows you to output basic HTML tags. This is typically used within renderer functions, for example `question/type/*pluginname*/renderer.php`.
+
+:::tip
+Please consider using [templates](../../../guides/templates/index.md), as an alternative to the _HTML writer_
+:::
+
+:::note
+
+There is no documentation for most of this class. Please look at the source code in `moodle/lib/outputcomponents.php`.
```

Comment:
Recommend pointing to this: https://phpdoc.moodledev.io/master/d4/d78/classhtml__writer.html